### The ugvs configuration file.
# cmdvel_topic: /car${id}/cmd_vel # geometry_msgs/Twist type
# mocap_pose_topic: /vrpn_client_node/car${id}/pose # geometry_msgs/PoseStamped type
# nluwb_pos_topic: /nlink_linktrack_anchorframe0 # nlink_parser/LinktrackAnchorframe0 type
# cfuwb_pos_topic: /car${id}/pos # geometry_msgs/PoseStamped type
# imu_topic: /car${id}/imu/data # sensor_msgs/Imu

ugvs:
  - id: 7
    pos_source: mocap # mocap, nluwb, cfuwb
    yaw_source: mocap # mocap, imu
    initialPosition: [0.25, 0.55, 0.0, -1.8] # [x(m),y(m),z(m),yaw(rad)]

  - id: 10
    pos_source: mocap # mocap, nluwb, cfuwb
    yaw_source: mocap # mocap or imu
    initialPosition: [0.0, 0.55, 0.0, 0.0] # [x(m),y(m),z(m),yaw(rad)]
  
  - id: 11
    pos_source: nluwb # mocap, nluwb, cfuwb
    yaw_source: imu # mocap, imu
    uwb_tag_id: 11
    initialPosition: [0.0, 0.0, 0.0, 1.57] # [x(m),y(m),z(m),yaw(rad)]